scripts/test

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env bash
-
-# If node 17 and higher, fail
-if [[ $(node -v) > "v17" ]]; then
-  echo "Using legacy provider"
-  export NODE_OPTIONS=--openssl-legacy-provider
-fi
-
+ååå
```

Comment:
Typo?

Reply:
Yes! Sorry, was on a plane preparing to land 🤦  I'll take a look at this again tonight.